lib/rgeo/active_record/arel_spatial_queries.rb

```diff
@@ -24,13 +24,14 @@ def st_func(standard_name)
 
       def visit_RGeo_ActiveRecord_SpatialNamedFunction(node, collector)
         name = st_func(node.name)
+        exprs = []
```

Comment:
In this case I think we are not limited by the line length etc. and we could use the full name `expressions` for this variable to improve the readability of the code